conans/client/build/cmake.py

```diff
@@ -232,6 +238,32 @@ def build_config(self):
             return "--config %s" % self._build_type
         return ""
 
+    def _get_standard_vars(self):
+        ret = {}
+        # Standard C++
+        if self._cppstd:
+            if self._cppstd.endswith("gnu"):
+                ret["CONAN_CMAKE_CXX_STANDARD"] = self._cppstd[:-3]
+                ret["CONAN_CMAKE_CXX_EXTENSIONS"] = "ON"
```

Comment:
Maybe CONAN_CMAKE_CXX_GNU_EXTENSIONS?

Reply:
the variable in CMake is `CMAKE_CXX_EXTENSIONS`:

```
set(CMAKE_CXX_EXTENSIONS OFF)
On some platforms, this results in linking to a different library (e.g. -std=c++11 rather than -std=gnu++11). The default behavior is for C++ extensions to be enabled. 
```